common/traits/wc_special_traits.txt

```diff
@@ -391,6 +391,11 @@ kiljaeden_trait = { #for portrait logic
 	random = no
```

Comment:
What is vrykul_male_base_12.dds and vrykul_male_hair_behind_12.dds?

Reply:
its old portraits i forgot to delete

Reply:
Deleted them.